invenio_accounts/ext.py

```diff
@@ -62,6 +63,8 @@ def init_app(self, app, use_celery=True):
             def delay_security_email(msg):
                 send_security_email.delay(msg)
 
+            FlaskCeleryExt(app)
```

Comment:
Yes, FlaskCeleryExt is being initialized in Invenio-Celery so this would cause double initialization.

Perhaps it's just missing in the example app?